src/tools/config/getFrontendConfig.ts

```diff
@@ -7,6 +7,22 @@ export const getFrontendConfig = (name: string): ConfigType => {
   const config = getConfig(name);
 
   Consola.info(`Requested frontend content of configuration '${name}'`);
+  // Find out if config as apps with integrations that have a property with no value or undefined
+  // If so, remove print an error using consola
+  // If not, return the config
+  const appsWithIntegrationsWithUndefinedProperties = config.apps.filter(
+    (app) =>
+      app.integration?.properties.some(
+        (property) => property.value === null || property.value === undefined
+      ) ?? false
+  );
+  if (appsWithIntegrationsWithUndefinedProperties.length > 0) {
+    Consola.warn(
+      `The following apps have integrations with errored properties: [${appsWithIntegrationsWithUndefinedProperties
```

Comment:
``errored properties`` - this is a really weird sentence IMO.
I would suggest ``There are apps, that do not contain credentials: <list>. Check if they are required``

Reply:
The point is that apps have properties with errors, it might not be related to credentials

Reply:
Okay, true. Perhaps ``There are apps, that have missing configuration options: <list>.``?